lib/sqlalchemy/util/compat.py

```diff
@@ -260,12 +259,6 @@ def formatargandannotation(arg):
 else:
     from inspect import formatargspec as inspect_formatargspec  # noqa
 
-if win32 or jython:
-    time_func = time.clock
```

Comment:
This is the fix for python3.8 - this time_func wasn't even used - and time.clock got deprecated long ago and was removed in py3.8